### PR TITLE
Update browser title for each page

### LINKED ISSUE #92: Correct page titles
Currently, all pages have a title "React App".
We need to set correct titles depends on a page.
Something like:
```
index: Source{d} Code Annotation Tool
experiment: <name> | Source{d} Code Annotation Tool
review: Review for experiment <name> | Source{d} Code Annotation Tool
```

### PR DESCRIPTION
**Pull request by carlosms**
Fixes #92 

Review.js does not work for now, but I hope it works after #94 is merged.

### CODE REVIEW

**Comment by smacker** [comment]
hm... Do you think it would be better to use https://github.com/nfl/react-helmet ?
it would allow to use templates for titles and instead of juggling with componentDidMount/componentWillReceiveProps you can just put it in render function.

also finish page is missed.

**Comment by carlosms** [comment]
Thanks @smacker, I've pushed new code using react-helmet.

File: src/pages/Experiment.js [hunk 1]
```diff
@@ -30,6 +31,9 @@ class Experiment extends Component {
 
     return (
       <div className="ex-page">
+        <Helmet>
+          <title>{`${this.props.name} | source{d} Code Annotation Tool`}</title>
```
Comment (smacker) [inline src/pages/Experiment.js]: please use `titleTemplate` to avoid copy-pasting suffix.

**Review by smacker — CHANGES_REQUESTED** [review]

**Review by dpordomingo — APPROVED** [review]

**Review by bzz — APPROVED** [review]
LGTM

**Review by smacker — APPROVED** [review]

**Comment by bzz** [comment]
@carlosms is there anything left before merging this guy?

**Pull request merged**